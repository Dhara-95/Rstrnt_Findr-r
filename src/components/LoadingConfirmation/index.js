import React from "react";

const LoadingConfirmation = () => {
  return <h1 className="text-grey">Loading...</h1>;
};

export { LoadingConfirmation };
