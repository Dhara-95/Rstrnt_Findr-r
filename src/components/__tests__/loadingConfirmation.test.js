import React from "react";
import renderer from "react-test-renderer";

import { LoadingConfirmation } from "../LoadingConfirmation";

it("displays Loading...", () => {
  const tree = renderer.create(<LoadingConfirmation />).toJSON();
  expect(tree).toMatchSnapshot();
});
