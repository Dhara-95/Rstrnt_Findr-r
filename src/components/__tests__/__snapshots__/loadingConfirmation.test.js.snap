// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays Loading... 1`] = `
<h1
  className="text-grey"
>
  Loading...
</h1>
`;
